setup.py

```diff
@@ -55,9 +96,7 @@
       install_requires=install_requires,
       package_data={'menpo': ['data/*',
                               'feature/cpp/*.cpp',
-                              'feature/cpp/*.h',
-                              'transform/piecewiseaffine/fastpwa/*.c',
-                              'transform/piecewiseaffine/fastpwa/*.h'],
-                    '': ['*.pxd', '*.pyx']},
+                              'transform/piecewiseaffine/fastpwa/*.cpp'],
+                    '': ['*.pxd', '*.pyx', '*.h']},
```

Comment:
can't we simplify this down now?
```diff
       package_data={'menpo': ['data/*',
-                              'feature/cpp/*.cpp',
-                              'transform/piecewiseaffine/fastpwa/*.cpp'],
-                    '': ['*.pxd', '*.pyx', '*.h']},
+                    '': ['*.pxd', '*.pyx', '*.cpp', '*.h']},
```
(i.e. we _always_ want `.cpp` files, just like `.pyx` etc)